src/auditwheel/_vendor/whichprovides/__init__.py

```diff
@@ -0,0 +1,359 @@
+# SPDX-License-Identifier: MIT
```

Comment:
This is a vendored project "whichprovides" which I maintain. It's the abstraction of the package provider detection that was mentioned here: https://github.com/pypa/auditwheel/issues/541#issuecomment-2656820738